jetty-ee9/jetty-ee9-servlet/src/test/java/org/eclipse/jetty/ee9/servlet/DefaultServletTest.java

```diff
@@ -362,10 +412,18 @@ public void testListingFilenamesOnlyUrlResource() throws Exception
     @Test
     public void testListingProperUrlEncoding() throws Exception
     {
-        ServletHolder defholder = context.addServlet(DefaultServlet.class, "/*");
-        defholder.setInitParameter("dirAllowed", "true");
-        defholder.setInitParameter("redirectWelcome", "false");
-        defholder.setInitParameter("gzip", "false");
+        Path docRoot = workDir.getEmptyPathDir().resolve("docroot");
+        FS.ensureDirExists(docRoot);
+
+        startServer((context) ->
+        {
+            context.setBaseResource(ResourceFactory.of(context).newResource(docRoot));
+
+            ServletHolder defholder = context.addServlet(DefaultServlet.class, "/*");
```

Comment:
I know it's this way in the original file, however the DefaultServlet is normally mapped at /, not /*

Reply:
Same as above, this is intentional, and historical, to match end user use cases.